frontend/src/common/components/MentoringApplicationStatus/MentoringApplicationStatus.tsx

```diff
@@ -6,85 +6,60 @@ import type { myTheme } from '../../types/theme';
 
 interface MentoringApplicationStatusProps {
   status: StatusType;
+  type?: 'CREATED' | 'PARTICIPATED';
 }
 
 const STATUS_DESCRIPTION = {
   [StatusTypeEnum.PENDING]: {
     VALUE: '승인대기',
-    EMOTICON: '⏳',
   },
   [StatusTypeEnum.APPROVED]: {
     VALUE: '승인됨',
-    EMOTICON: '✅',
   },
   [StatusTypeEnum.COMPLETE]: {
     VALUE: '완료됨',
-    EMOTICON: '🎉',
   },
   [StatusTypeEnum.REJECTED]: {
     VALUE: '거절됨',
-    EMOTICON: '❌',
+  },
+} as const;
+
+const STATUS_PARTICIPATED_DESCRIPTION = {
+  [StatusTypeEnum.PENDING]: {
+    VALUE: '예약신청',
+  },
+  [StatusTypeEnum.APPROVED]: {
+    VALUE: '예약확정',
+  },
+  [StatusTypeEnum.COMPLETE]: {
+    VALUE: '완료됨',
+  },
+  [StatusTypeEnum.REJECTED]: {
+    VALUE: '거절됨',
   },
 } as const;
 
 function MentoringApplicationStatus({
   status,
+  type = 'CREATED',
 }: MentoringApplicationStatusProps) {
+  const description =
+    type === 'CREATED'
+      ? STATUS_DESCRIPTION[status]
+      : STATUS_PARTICIPATED_DESCRIPTION[status];
```

Comment:
p5: 만약 STATUS_DESCRIPTION에 모아두면 삼항연산자 대신에 이렇게 할 수 있을 것 같아!

```suggestion
const description = STATUS_DESCRIPTION[type][status];
```

Reply:
여기까지 신경써서 수정해줬구나 고마워 ㅎㅎ